add dmn modeling page to web modeler docs

SebastianStamm opened the pull request.

CatalinaMoisuc commented:
FYI @akeller I discussed with @MaxTru and he will change [this PR](https://github.com/camunda-cloud/camunda-cloud-documentation/pull/738) to make the guide under DMN generic as he did with [this one](https://github.com/camunda-cloud/camunda-cloud-documentation/pull/739).

I also looked into the [save and deploy](https://github.com/camunda-cloud/camunda-cloud-documentation/blob/web-modeler-dmn/docs/components/modeler/web-modeler/save-and-deploy.md) page within the context of DMN and I think it looks fine and applies to both DMN and BPMN since we don't call them by name but simply "diagram".


So assuming we have a generic DMN page, how much content from [this page](https://github.com/camunda-cloud/camunda-cloud-documentation/blob/web-modeler-dmn/docs/components/modeler/web-modeler/modeling-dmn.md) in the Web Modeler section we need? Wdyt @akeller ?

akeller commented:
> So assuming we have a generic DMN page for both Modelers, how much content from [this page](https://github.com/camunda-cloud/camunda-cloud-documentation/blob/web-modeler-dmn/docs/components/modeler/web-modeler/modeling-dmn.md) in the Web Modeler section we need? Wdyt @akeller ?

I have not had a chance to go hands on with the modeling components, but if the actions for modeling DMN are identical (which is looks like they are), let's have one page to cover the experience in desktop and web modeler.

The only difference we may want to document is the auto-save feature on web modeler, but this could be done with a simple admonition/note.

akeller commented:
Let me know if you need more clarification from me here. I merged @MaxTru's other PRs.

CatalinaMoisuc commented on docs/components/modeler/web-modeler/modeling-dmn.md:
```diff
@@ -0,0 +1,49 @@
+description: Let's create a DMN diagram.
+---
+
+## Create new DMN decision requirement diagram
```
@SebastianStamm then let's move this one to `launch-cloud-modeler` and mention there that you can create DMN and BPMN.

CatalinaMoisuc replied:
So essentially after this part is moved, we should remove the page with all the screenshots.

CatalinaMoisuc commented on docs/components/modeler/web-modeler/modeling-dmn.md:
```diff
@@ -0,0 +1,49 @@
+
+## Start modeling
+
+![Start Modeling](./img/dmn/main.png)
```
The rest should be covered as @akeller mentioned by @MaxTru's PR: https://github.com/camunda-cloud/camunda-cloud-documentation/pull/738#pullrequestreview-932578226

CatalinaMoisuc commented:
> > So assuming we have a generic DMN page for both Modelers, how much content from [this page](https://github.com/camunda-cloud/camunda-cloud-documentation/blob/web-modeler-dmn/docs/components/modeler/web-modeler/modeling-dmn.md) in the Web Modeler section we need? Wdyt @akeller ?
> 
> I have not had a chance to go hands on with the modeling components, but if the actions for modeling DMN are identical (which is looks like they are), let's have one page to cover the experience in desktop and web modeler.
> 
> The only difference we may want to document is the auto-save feature on web modeler, but this could be done with a simple admonition/note.

Thanks for clarifying @akeller. Then let's proceed like this. We only mention DMN  where needed in the existent documentation (i.e. create, import etc.) and we keep the common DMN modeling page that Max proposed.

SebastianStamm commented:
@CatalinaMoisuc I have removed the dedicated web-modeler DMN page together with the screenshots and adjusted the "Launch Web Modeler" page to include DMN and Forms as well as an updated screenshot. Please let me know if I need to make any other changes.

The failing CI is unrelated to the changes in this PR.

CatalinaMoisuc reviewed (approved):
lgtm.

CatalinaMoisuc commented:
@akeller do you have any idea about the broken links reported in [the build](https://github.com/camunda-cloud/camunda-cloud-documentation/runs/5848914161?check_suite_focus=true)?

akeller commented:
I'll take care of it @CatalinaMoisuc 😄

akeller reviewed (approved):
LGTM! And fixed the build issue by pulling in the latest release branch changes 😄
